package/cfg/config.yaml

```diff
@@ -188,6 +199,7 @@ version_mapping:
   "rke-1.16": "rke-cis-1.5-permissive"
   "rke-1.17": "rke-cis-1.5-permissive"
   "rke-1.18": "rke-cis-1.5-permissive"
+  "rke-1.18+": "rke2-cis-1.5"
```

Comment:
We will also need to update the CIS v2 chart with corresponding ClusterScanBenchmark and ClusterScanProfile to use this version